packages/terra-dialog/tests/jest/Dialog.test.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import Dialog from '../../src/Dialog';
+
+describe('Dialog', () => {
+  const defaultRender = <Dialog />;
+
+  // Snapshot Tests
+  it('should render a default component', () => {
+    const wrapper = shallow(defaultRender);
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  // Prop Tests
+  it('should use the default value when no value is given', () => {
+    const wrapper = shallow(defaultRender);
+    expect(wrapper.find('.dialog').text()).toEqual('defualt');
```

Comment:
*default. Update the jest snapshots and we should be good.

Reply:
https://github.com/cerner/terra-core/pull/1158/commits/f7c833de381df9ada04ac26c89e11b89c660355e